app/src/main/res/values/strings.xml

```diff
@@ -36,5 +36,8 @@
 
     <string name="search_bar_hint_text">Search</string>
     <string name="search_result">No search results found.</string>
-
+    <string name="shop_web_site">Web Site</string>
+    <string name="shop_address">Address:</string>
+    <string name="shop_phone_number">Phone:</string>
```

Comment:
It would be better to write Address: %s as a single, spannable text, rather than using two separate text elements in a row.

Reply:
And the same for the phone